code/modules/cargo/bounties/item.dm

```diff
@@ -0,0 +1,35 @@
+/datum/bounty/item
+	var/required_count = 1
+	var/shipped_count = 0
+	var/list/wanted_types = list()  // Types accepted for the bounty.
+	var/include_subtypes = TRUE     // Set to FALSE to make the datum apply only to a strict type.
+	var/list/exclude_types = list() // Types excluded.
```

Comment:
Since this is mostly unused, if you can leave it uninitialized to any value you can save a few bytes of memory

Reply:
I think it's worth keeping. A few wasted bytes isn't much concern, but the chance of introducing runtimes or related bugs down the line is.

Reply:
these actually cost runtime performance because they cause an init.

leave it as a null list